examples/terraform-submodules/gke/module.tf

```diff
@@ -47,17 +47,44 @@ variable "node_count" {
   default = "4"
 }
 
-module "agones" {
-  source = "git::https://github.com/googleforgames/agones.git//install/terraform/?ref=master"
+module "gke_cluster" {
+  // ***************************************************************************************************
+  // Update ?ref= to the agones release you are installing. For example, ?ref=release-1.3.0 corresponds
+  // to Agones version 1.3.0
+  // ***************************************************************************************************
+  source = "git::https://github.com/googleforgames/agones.git//install/terraform/modules/gke/?ref=master"
 
   cluster = {
     "zone"             = "us-west1-c"
     "name"             = var.name
     "machineType"      = var.machine_type
     "initialNodeCount" = var.node_count
     "project"          = var.project
+    "network"          = "default"
```

Comment:
Were you planning on changing this to a var and setting the default in the variable instead?

Reply:
Good call. Done.